package.json

```diff
@@ -18,58 +18,44 @@
   "main": "none",
-  "workspaces": [
-    "api-server",
-    "client",
-    "client/plugins/fcc-source-challenges",
-    "client/plugins/gatsby-remark-node-identity",
-    "curriculum",
-    "tools/challenge-editor/api",
-    "tools/challenge-editor/client",
-    "tools/challenge-helper-scripts",
-    "tools/challenge-parser",
-    "tools/scripts/build",
-    "tools/scripts/seed",
-    "tools/ui-components"
-  ],
   "scripts": {
-    "audit-challenges": "npm run create:config && ts-node tools/challenge-auditor/index.ts",
+    "audit-challenges": "pnpm run create:config && ts-node tools/challenge-auditor/index.ts",
     "analyze-bundle": "webpack-bundle-analyzer",
     "prebuild": "npm-run-all create:*",
     "build": "npm-run-all -p build:*",
-    "build-workers": "cd ./client && npm run prebuild",
-    "build:client": "cd ./client && npm run build",
-    "build:curriculum": "cd ./curriculum && npm run build",
-    "build:server": "cd ./api-server && npm run build",
+    "build-workers": "cd ./client && pnpm run prebuild",
+    "build:client": "cd ./client && pnpm run build",
+    "build:curriculum": "cd ./curriculum && pnpm run build",
+    "build:server": "cd ./api-server && pnpm run build",
     "challenge-editor": "npm-run-all -p challenge-editor:*",
     "challenge-editor:client": "cd ./tools/challenge-editor/client && npm start",
     "challenge-editor:server": "cd ./tools/challenge-editor/api && npm start",
     "clean": "npm-run-all -p clean:*",
-    "clean-and-develop": "npm run clean && npm ci && npm run develop",
-    "clean:client": "cd ./client && npm run clean",
+    "clean-and-develop": "pnpm run clean && pnpm install && pnpm run develop",
+    "clean:client": "cd ./client && pnpm run clean",
     "clean:curriculum": "npx --yes rimraf ./config/curriculum.json",
     "clean:packages": "npx --yes rimraf ./**/node_modules",
     "clean:server": "npx --yes rimraf ./api-server/lib",
-    "create:config": "tsc -p config && npm run ensure-env && npm run download-trending",
+    "create:config": "tsc -p config && pnpm run ensure-env && pnpm run download-trending",
     "create:utils": "tsc -p utils",
     "precypress": "node ./cypress-install.js",
     "cypress": "cypress",
-    "cypress:dev:run": "npm run cypress -- run --spec cypress/e2e/default/**/*.js",
-    "cypress:dev:watch": "npm run cypress -- open",
-    "cypress:prd:run": "npm run cypress -- run --spec cypress/e2e/default/**/*.js",
-    "cypress:prd:watch": "npm run cypress -- open",
+    "cypress:dev:run": "pnpm run cypress -- run --spec cypress/e2e/default/**/*.js",
+    "cypress:dev:watch": "pnpm run cypress -- open",
+    "cypress:prd:run": "pnpm run cypress -- run --spec cypress/e2e/default/**/*.js",
+    "cypress:prd:watch": "pnpm run cypress -- open",
     "cypress:install": "cypress install && echo 'for use with ./cypress-install.js'",
     "cypress:install-build-tools": "sh ./cypress-install.sh",
     "predevelop": "npm-run-all create:*",
```

Comment:
Is it still fine to use `npm-run-all`?

Reply:
Weirdly, yes: https://github.com/pnpm/pnpm/issues/413